semver.md

```diff
@@ -319,6 +319,22 @@ Abbreviating "version" as "v" is often seen with version control. Example:
 `git tag v1.2.3 -m "Release version 1.2.3"`, in which case "v1.2.3" is a tag
 name and the semantic version is "1.2.3".
 
+### What does SemVer consider a package?
+
+The meaning of package in this specification describes a deployment unit.
+It could be built from a single or multiple source artifacts. But it can
+also be an assembly of multiple binary artifacts.
+
+Semantic versioning concepts can however also be applied to logical
+collections like an interface, an API or any other concept which
+undergoes changes and can be assigned a version.
+
+In Java for example a typical deployment unit would be a JAR file and a
+Java package is typically not versioned independenly.
+However with modular technologies (like *OSGi bundles*) the Java package
```

Comment:
Very minor nit: this sentence should start on the line above.

Reply:
Do you mean breaking the line after ‚However with...‘? 

(i can do that, I am just used to breaking lines by sentence for markup, which makes generally more stable diffs. But I will try to fit in the existing style)

Reply:
Correction: will line join it as well

Reply:
> "which makes generally more stable diffs"

Oh, I hadn't thought of that. Like I said, it was a nit, perhaps you should ignore me on this one.

Reply:
Relevant: http://rhodesmill.org/brandon/2012/one-sentence-per-line/